### PR TITLE
Shim workaround for livepatch tests

### PR DESCRIPTION
**Pull request by mdoucha**
Kernels released before Boothole fixes will no not boot with the latest shim on machines with SecureBoot enabled. Override shim version if needed to allow livepatch tests to boot.

- Related ticket: N/A
- Needles: N/A
- Verification runs:
  - 12-SP5: https://openqa.suse.de/tests/4925198
  - 15GA: https://openqa.suse.de/tests/4925199
  - 15-SP1: https://openqa.suse.de/tests/4925200
  - 15-SP2: https://openqa.suse.de/tests/4925201
  - Post-Boothole: https://openqa.suse.de/tests/4925202
  - Non-SecureBoot: https://openqa.suse.de/tests/4925203

### CODE REVIEW

**Comment by mdoucha** [comment]
All verification runs finished with expected test failures only. This PR is ready for merge.

**Review by ggkioulis — APPROVED** [review]
lgtm! nice work with `package_version_cmp`

File: lib/version_utils.pm [hunk 1]
```diff
@@ -652,3 +653,42 @@ sub has_test_issues() {
...
+
+=head2 package_version_cmp
+
+Compare two SUSE-style version strings. Returns an integer that is less than,
+equal to, or greater than zero if the first argument is less than, equal to,
+or greater than the second one, respectively.
+
+=cut
+
+sub package_version_cmp {
```
Comment (foursixnine) [inline lib/version_utils.pm:665]: Mind adding tests for this specifically? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/t/01_version_utils.t

Comment (mdoucha) [reply lib/version_utils.pm:665]: Unit test added.

**Review by foursixnine — APPROVED** [review]

**Pull request merged**